### ISSUE TITLE
Ability to sync with other calendars

### ISSUE THREAD

**Issue by MTRNord** [nordeck/matrix-meetings#515]
**Is your feature request related to a problem? Please describe.**

External meetings often come in via Teams or other places. It would be nice to have a 2 way sync via webdav or at least 1 way sync via ics with other calendars to make sure there is no confusion about which calendar to use and to easy seeing conflicts.

**Describe the solution you'd like**

Specifically, in my usecase I run a radicale webdav as a single place of truth for all my calendars. A way to push the calendar entries from the bot into it would be nice.

An MVP however would be ics export and ics import, for example for the outlook/teams calendars. (Outlook has a share calendar endpoint to share the calendar of your user via an ics file containing all your calendar events, but they don't have a way to upload it without their API)

**Describe alternatives you've considered**

Manually matching of course works, but is prone to cause errors.

**Additional context**

This is a request privately. While it would help at work, it's not project driven currently :)

**Comment by charlynguyen** [comment]
Thanks for the report! 🙂 We picked up the issue. Updates will be shared here.

Feel free to contribute 😉

**Comment by rasos** [comment]
Yes it would be very useful to add a CalDav server so that the calendar can be synced e.g. with a Nextcloud calendar. Also the other way round, adding a CalDav client, would be useful for some cases, however Nextcloud does not offer yet a Caldav server, see discussion at https://github.com/nextcloud/calendar/issues/2766#issuecomment-2019396113 .

**Comment by ibotty** [comment]
It might make sense to look into pimsync and "simply" add a matrix-meetings backend there.  That way you'll get other people to take care of the idiosyncrasies of CalDav and related protocols.
